contrib/src/databases.rs

```diff
@@ -0,0 +1,65 @@
+use rocket::Rocket;
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::config::{Config, Table, Value};
+use rocket::config::Value::String;
+
+pub struct Databases;
+
+#[cfg(feature = "diesel_sqlite")]
+use diesel::SqliteConnection;
+use diesel::r2d2::ConnectionManager;
+
+#[cfg(feature = "pg_pool")]
+use r2d2_postgres::{PostgresConnectionManager, TlsMode};
+
+#[cfg(any(feature = "diesel_sqlite", feature = "pg_pool"))]
+use r2d2::Pool;
+
+// TODO: Assemble methods to instantiate connection pool based on configured options
+impl Databases {
+    pub fn fairing() -> Self {
+        Databases
+    }
+
+    #[cfg(feature = "diesel_sqlite")]
+    pub fn init_sqlite_pool<'a>(&self, credentials: &'a str) -> Pool<ConnectionManager<SqliteConnection>> {
+        let manager = ConnectionManager::<SqliteConnection>::new(credentials);
+        Pool::new(manager).expect("Unable to create sqlite pool")
+    }
+
+    #[cfg(feature = "pg_pool")]
+    pub fn init_rust_pg_pool<'a>(&self, credentials: &'a str) -> Pool<PostgresConnectionManager> {
+        let manager = PostgresConnectionManager::new(credentials, TlsMode::None).unwrap();
+        Pool::new(manager).expect("Unable to create postgres pool")
+    }
+}
```

Comment:
I think using `#[cfg(feature = "...")]` works to only compile the symbols we know we'll have in scope. This is my preferred and first approach. If there's a better one, I'd be more than happy to explore it.

Reply:
Yeah, I think that's fine. Can we name features/adapters with a `/`? If so, I think that would work nicely. Then the user can ask for `diesel/sqlite`.

Reply:
Sounds good. I'll keep this as the approach we use going forward.